src/main/resources/templates/email/TechnicalQuestions_Interviewer.txt

```diff
@@ -0,0 +1,14 @@
+Hello {{interviewer_first_name}},
+
+We see that you will be giving a {{position}} mock interview with InterviewMe on {{formatted_date}}. These interviews usually come with technical questions, which you should use to measure the candidate's competency. We wanted to share with you some tips on handling this technical portion.
+
+  - Leave some vagueness in the question. This gives the candidate the opportunity to ask clarifying questions.
+  - Does the candidate have the right approach in mind? Or is theirs going to lead them in the wrong direction? If you feel they are on the wrong track, hint them so.
+  - Does the candidate make progress through the question? If they haven't been progressing, maybe it's time for a hint. 
+  - Remember, this is a conversation, not a monologue.
+  - If the candidate doesn't do so, prompt them to consider run times of proposed solutions.
+  - After coding, if the candidate doesn't do so, work through a small example to try and catch any logic/errors.
```

Comment:
Again, this should be throughout, and an interviewer should suggest using it whenever it is unclear whether a step that was taken works.

Reply:
See above comment.